core/src/main/java/io/aiven/klaw/error/KlawExceptionHandler.java

```diff
@@ -72,10 +64,6 @@ protected ResponseEntity<Object> handleMethodArgumentNotValid(
       WebRequest request) {
     log.error("Validation Error ", ex);
     return new ResponseEntity<>(
```

Comment:
I dont see this method actually being used I think we can delete it.

Reply:
the issue is that it overrides `org.springframework.web.servlet.mvc.method.annotation.ResponseEntityExceptionHandler#handleMethodArgumentNotValid`
so it could be invoked by spingframework